qiskit_nature/circuit/library/ansaetze/excitation_builder.py

```diff
@@ -0,0 +1,115 @@
+# This code is part of Qiskit.
+#
+# (C) Copyright IBM 2021.
+#
+# This code is licensed under the Apache License, Version 2.0. You may
+# obtain a copy of this license in the LICENSE.txt file in the root directory
+# of this source tree or at http://www.apache.org/licenses/LICENSE-2.0.
+#
+# Any modifications or derivative works of this code must retain this
+# copyright notice, and modified files need to carry a notice indicating
+# that they have been altered from the originals.
+"""
+TODO.
+"""
+
+from typing import List, Tuple
+
+import itertools
+import logging
+
+from qiskit_nature.operators.second_quantization import FermionicOp, SecondQuantizedOp
+
+logger = logging.getLogger(__name__)
+
+
+class ExcitationBuilder:
```

Comment:
If this only provides a single staticmethod, can we make it a callable instead of a class? I think this will be simpler to use and users are not exposed to even more classes 🙂